### ISSUE TITLE
docs/getting started

### ISSUE THREAD

**Issue by step21** [sat-metalab/scenic#79]
Hi, so finally scenic is running - but the getting started on github seems a bit outdated/incomplete, and I couldn't find any other docs to get started. Is there some easy way to get started? Is it relatively simple to stream from one machine with say, RTP, to either vlc client or another scenic machine, or do I need an intermediary server? Does scenic also handles NAT translation or some ports being blocked by firewalls?

**Comment by step21** [comment]
Some more notes - usage refers to https://github.com/sat-metalab/SCENIC/wiki but that is unfortunately quite empty. Scenic clearly has changed since Getting Started/Premier Pas was written, is this still current?
Maybe @holodny could weigh in as they created a wiki page 'scenic notes'

**Comment by nicobou** [comment]
Hi, we have planned to write a getting started. Since we are currently into a major UI update, documentation has been postponed until more stability. 
However I can answer some of your questions here:
- SIP/RTP transmission through NAT is available, you need to install [https://github.com/sat-metalab/scenic-server](a SIP server configured for scenic). This is the preferred way to transmit streams. 
- sending RTP to vlc can be achieved but only video on only audio, both together in the same stream did not work when it has been last tried something like two years ago.

**Comment by step21** [comment]
Thanks for your reply. And thanks for this software, now it just has to work for us :) Thanks for taking the time to explain and giving some hints on where to investigate/go next, it is very much appreciated.

**Comment by ubald** [comment]
Hi, this issue has been migrated to gitlab at https://gitlab.com/sat-metalab/scenic/issues/79
